lib/WTSI/NPG/HTS/PacBio/Sequel/AnalysisPublisher.pm

```diff
@@ -242,7 +246,9 @@ sub _get_tag_from_fname {
     my ($bc1, $bc2) = ($1, $2);
     $tag_id = ($bc1 == $bc2) ? $bc1 : undef;
   }
-  defined $tag_id or $self->logcroak("No tag found for $file");
+
+  defined ($tag_id || $file =~ /$NOT_DEPLEXED/smx) or
```

Comment:
I think this is a bit of a tricksy way of expressing the test - is this now relying on `0` evaluating `false` proceed to test the file name for `$NOT_DEPLEXED`, whereas before the change, tag `0` would evaluate `true` (because of `defined`). I think the logic here needs some comments for those not familiar with the tag `0` process for Sequel (like me!)

Reply:
Some additional documentation added to WTSI::NPG::HTS::PacBio::Sequel::AnalysisPublisher as requested.